index.html

```diff
@@ -3,28 +3,51 @@
 
 <head>
  <Title>Date Range Analyser</Title>
- <script src="./index.js">
- </script>
+
 
  <style>
       .header {
         padding-top: 10px;
         padding-bottom: 10px;
+        text-decoration: underline;
+      }
+      .bodySection {
+        font-size: 40px;
       }
       .Button {
-        margin-top:25px;
+        margin-top:45px;
+        background-color: #D9D3BB;
+        border: 4px;
+        border-color: black;
+        padding: 15px 32px;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+        font-size: 16px;
+        margin: 4px 2px;
+        cursor: pointer;
+        border-top: 10px;
 
       }
 
       .Start_Date {
```

Comment:
I would stay consistent in your naming of variables - i.e. using camel case everywhere or underscores everywhere. You already have "bodySection" as a class name elsewhere - I would stick to camelCase.

Reply:
Edited code for variable naming consistency.